bin/test_travis.sh

```diff
@@ -100,13 +101,16 @@ ${CXX} -std=c++0x $compile_flags expand1.cpp $link_flags
 export LD_LIBRARY_PATH=$our_install_dir/lib:$LD_LIBRARY_PATH
 ./a.out
 
-echo "Checking whether all header files are installed:"
-python $SOURCE_DIR/symengine/utilities/tests/test_make_install.py $our_install_dir/include/symengine/ $SOURCE_DIR/symengine
-
+echo $SOURCE_DIR
+cd $SOURCE_DIR
 # check trailing whitespace:
-if !  egrep " $" -nr --include=\*.{cpp,h,inc}  --exclude-dir=*{teuchos,/build/}* $SOURCE_DIR ; then
+if !  egrep " $" -nr --include=\*.{cpp,h,inc}  --exclude-dir=*teuchos* --exclude-dir=*build* $SOURCE_DIR ; then
     echo No trailing whitespace;
 else
+    echo Please remove above stated whitespaces;
```

Comment:
Why aren't the `echo` statements visible here https://travis-ci.org/symengine/symengine/jobs/112215038 ?